test/client/api.js

```diff
@@ -2,599 +2,601 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-var EventEmitter = require('events').EventEmitter
-var util = require('util')
+module.exports = config => {
```

Comment:
The rest of this module looks like it has changed but it's only indentation. You may want to [give it some `?w=1`](https://github.com/mozilla/fxa-auth-server/pull/1470/files?w=1).